variables.tf

```diff
@@ -14,75 +14,82 @@
  * limitations under the License.
  */
 
-variable project {
+variable "project" {
   description = "The project to deploy to, if not set the default provider project is used."
   default     = ""
 }
 
-variable region {
+variable "region" {
   description = "Region for cloud resources."
   default     = "us-central1"
 }
 
-variable network {
+variable "network" {
   description = "Name of the network to create resources in."
   default     = "default"
 }
 
-variable subnetwork {
+variable "subnetwork" {
   description = "Name of the subnetwork to create resources in."
   default     = "default"
 }
 
-variable network_project {
+variable "network_project" {
   description = "Name of the project for the network. Useful for shared VPC. Default is var.project."
   default     = ""
 }
 
-variable name {
+variable "name" {
   description = "Name for the forwarding rule and prefix for supporting resources."
 }
 
-variable backends {
+variable "backends" {
   description = "List of backends, should be a map of key-value pairs for each backend, must have the 'group' key."
-  type        = "list"
+  type        = list(object({ group = string }))
```

Comment:
The `description` key is also supported.

Reply:
Yes, it's supported, but should it be required?

Reply:
Fixed